src/commands/linkArticles.ts

```diff
@@ -83,7 +86,42 @@ async function insert(
 	if (templateString.includes("{description}") && !article.description)
 		new Notice("The article has no description.");
 
+	if (template.createArticleNote) {
+		if (!template.createArticleNoteUseCustomPath) {
+			template.createArticleNoteCustomPath = settings.createArticleNotePath;
+		}
+		const createdNote = await createNoteInFolder(app, noteTitle, insert, template);
+		if (createdNote != null) {
+			insert = `[[${createdNote}|${noteTitle}]]`;
+		}
+	}
+
 	const cursorPosition = editor.getCursor();
 	editor.replaceSelection(insert);
 	if (settings.placeCursorInfrontOfInsert) editor.setCursor(cursorPosition);
 }
+
+async function createNoteInFolder(
+	app: App,
+	title: string,
+	content: string,
+	template: Template
+): Promise<string | null> {
+	const newNotePath = normalizePath(`${template.createArticleNoteCustomPath}/${title}.md`);
+	const existingFile = app.vault.getAbstractFileByPath(newNotePath);
+	
+	if (existingFile) {
+		new Notice(`File already exists in the current folder. Instead directly linked article on cursor/selection.`);
```

Comment:
This notice should match my previous request regarding lines 89 to 101. Also drop the "current"

Reply:
You mean the one with the normalizedTitle variable in the message?

Reply:
I don't understand. I mean the one I highlighted in line 114. My other request says that if no note could be created the plugin should do nothing instead of bringing the content that was meant for the note into the current file. If this behavior is implemented this notice needs to reflect that. That's what I mean. Do you understand?

Reply:
Ok now I understood, what you meant by your previous reqeust. Yeah I`ll change that as well.